charts/pega/charts/installer/templates/_pega-installer-job.tpl

```diff
@@ -40,6 +40,11 @@ spec:
       serviceAccountName: {{ .root.Values.serviceAccountName }}
 {{- end }}   
       volumes:
+{{- if and .root.Values.installerMountVolumeClaimName (eq .root.Values.upgrade.automaticResumeEnabled "true") }}
+      - name: {{ template "pegaInstallerMountVolume" }}
+        persistentVolumeClaim:
+          claimName: {{ .root.Values.installerMountVolumeClaimName }}
+{{- end }}
```

Comment:
Should we just mount the volume if provided? Is there a need to check for `automaticResumeEnabled `?

Reply:
Addressed